python/ribasim/ribasim/node.py

```diff
@@ -25,9 +25,23 @@
     "ManningResistance": "D",
     "TabulatedRatingCurve": "D",
     "Pump": "h",
+    "Terminal": "s",
     "": "o",
 }
 
+_COlORS = {
```

Comment:
1. I assume the `l` became lowercase by accident
2. How about you name it `COLORS` (without the `_` prefix) and inline it in the `plot` function, since it is only used there?